.github/workflows/build-pr-preview.yml

```diff
@@ -0,0 +1,39 @@
+name: Build PR preview
```

Comment:
⚠️ I don't understand this workflow: currently this is a mix of preview deployment and "build site" (xref validation)

Your component has no dependencies on other component so, instead, you should have something like https://github.com/bonitasoft/bonita-labs-doc/blob/b758954b2b9343f8f75e2548ba0f8f8ab44a930e/.github/workflows/build-pr-preview.yml

Reply:
I guess the mix is because a lot of things are not yet clear in my head and it's a mix of copy-paste/trying to workout how exactly everything works. But I'll keep it simple and not build multiple projects atm. Thank you :)

Reply:
fixed